Makefile

```diff
@@ -12,7 +12,7 @@ protobuf:
 
 protobuf_docker:
 	mkdir -p ${PROTO_GEN_DIR}
-	docker run --rm -v "${PWD}":/protobuf -w="/protobuf" cyborgthefirst/grpc-swift:1.0.0-alpha.9 \
+	docker run --rm -v "${PWD}":/protobuf -w="/protobuf" cyborgthefirst/grpc-swift:1.0.0-alpha.11 \
```

Comment:
@cyborgthefirst I've updated Makefile, but this image tag doesn't exist yet in Docker Hub.

Reply:
Uploaded it right now, so it should work now.

Reply:
Thanks, that was quick!